clang/lib/Basic/TargetInfo.cpp

```diff
@@ -560,11 +560,12 @@ ParsedTargetAttr TargetInfo::parseTargetAttr(StringRef Features) const {
     }
 
     // While we're here iterating check for a different target cpu.
-    if (Feature.startswith("arch=")) {
+    if (Feature.startswith("arch=") || Feature.startswith("cpu=")) {
```

Comment:
We shouldn't be allowing 'cpu' in x86/x86-64 mode, as this is supposed to be compatible with GCC and I don't see value in it as a non-compatible extension.